openedx/core/djangoapps/call_stack_manager/tests.py

```diff
@@ -0,0 +1,238 @@
+"""
+Test cases for Call Stack Manager
+"""
+from mock import patch
+from django.db import models
+from django.test import TestCase
+
+from openedx.core.djangoapps.call_stack_manager import donottrack, CallStackManager, CallStackMixin
+from openedx.core.djangoapps.call_stack_manager.core import Glb
+
+
+class ModelMixinCSM(CallStackMixin, models.Model):
```

Comment:
Maybe `ModelMixinCallStackMgr` instead of `ModelMixinCSM`? And below.

The acronym conflicts with the other CSM...